x-pack/test/api_integration/apis/uptime/rest/helpers/expect_fixture_eql.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import fs from 'fs';
+import { join } from 'path';
+import { cloneDeep } from 'lodash';
+
+const fixturesDir = join(__dirname, '..', 'fixtures');
+
+const excludeFieldsFrom = (from: any, excluder?: (d: any) => any): any => {
+  const clone = cloneDeep(from);
+  if (excluder) {
+    excluder(clone);
+  }
+  return clone;
+};
+
+export const expectFixtureEql = <T>(data: T, fixtureName: string, excluder?: (d: T) => void) => {
```

Comment:
Do we need to duplicate this? Couldn't we just configure the existing function to accept a default/alternative `path` parameter, or is there some other reason for this? Happy to chat about this if I'm missing something or we need clarification.

Reply:
done !!